api/controllers/UserController.js

```diff
@@ -139,13 +62,13 @@ module.exports = {
       });
   },
 
+  // route redirects requests (not page) to "read" auto-created CRUD route of current user
   currentuser: function(req, res) {
```

Comment:
@mmomen 
This defines the behavior of one of the routes, and I think our routes aren't set be case sensitive but I'm not sure.